tests/unittest.h

```diff
@@ -11,7 +11,7 @@
 void testAssertionFailed(const char *message, const char *file, int line) {
     fprintf(stderr, "Assertion failed in file: %s on line: %d\n", file, line);
     if (strlen(message) > 0) {
```

Comment:
I guess this check should: `if (message && strlen(message) > 0)` - so that you can safely pass `NULL` as message argument.